### PR TITLE
file-upload doc and image

### PR DESCRIPTION
**Pull request by burnout87**

### CODE REVIEW

**Review by dsavchenko — APPROVED** [review]

**Review by dsavchenko — APPROVED** [review]

File: content/docs/guide-development.md [hunk 1]
```diff
@@ -120,7 +121,31 @@ if credentials_env:
 	credentials=json.loads(credentials_env)
 	client = Client(address=credentials["address"])
 
-``` 
+```
+
+### How to upload a file to be used for the notebook execution
+
+In order to use a file during the notebook execution, two possible annotations (from the MMODA [ontology](https://odahub.io/ontology/)) are available:
```
Comment (volodymyrss) [inline content/docs/guide-development.md:128]:
```suggestion
In order to define an file as one of the input notebook parameters, two possible annotations (from the MMODA [ontology](https://odahub.io/ontology/)) are available:
```

File: content/docs/guide-development.md [hunk 2]
```diff
@@ -120,7 +121,31 @@ if credentials_env:
...
+
+* `oda:POSIXPath`
+* `oda:FileURL`
+
+#### `oda:PosixPATH`
```
Comment (volodymyrss) [inline content/docs/guide-development.md:133]:
```suggestion
#### `oda:POSIXPath`
```
?

File: content/docs/guide-development.md [hunk 3]
```diff
@@ -120,7 +121,31 @@ if credentials_env:
...
+
+This is used to reference a file locally available in the repository. For example:
+
+```python
+img_file_path = 'image.png' # oda:POSIXPath
+```
+
+#### `oda:FileURL`
+
+This is instead to have a url of a file. For example:
```
Comment (volodymyrss) [inline content/docs/guide-development.md:143]:
```suggestion
This annotation is instead used when the notebook expects parameter to be a URL of a file. For example:
```

File: content/docs/guide-development.md [hunk 4]
```diff
@@ -120,7 +121,31 @@ if credentials_env:
...
+* `oda:FileURL`
+
+#### `oda:PosixPATH`
+
+This is used to reference a file locally available in the repository. For example:
+
+```python
+img_file_path = 'image.png' # oda:POSIXPath
+```
+
```
Comment (volodymyrss) [inline content/docs/guide-development.md:140]:
```suggestion
It is advisable to put an example file in the repository matching the default file name ( `'image.png'`  in the example above) to allow easy testing of the notebook.
```

File: content/docs/guide-development.md (hunk 2 above)
Comment (burnout87) [reply content/docs/guide-development.md:133]: true...

**Review by volodymyrss — APPROVED** [review]

**Comment by ferrigno** [comment]
I added two sentences to explain what will appear in the frontend, check if this is correct.

**Review by ferrigno — APPROVED** [review]

**Comment by ferrigno** [comment]
I see that it is correct, it is worth making a repetition.

**Pull request merged**